# Deprecated
---
osa_msgs/MotorCmdMultiArray motor_cmd_multi_array
